apps/asap-server/scripts/create-algolia-backup.js

```diff
@@ -0,0 +1,28 @@
+const algoliasearch = require('algoliasearch');
+const fs = require('fs/promises');
+
+const main = async () => {
+  console.log({
+    appId: process.env.ALGOLIA_APP_ID,
+    apiKey: process.env.ALGOLIA_API_KEY,
```

Comment:
Should we be revealing secrets in logs?

Reply:
probably should also include ALGOLIA_INDEX

Reply:
we should not reveal them, no.

As for the TS - seems like an unnecessary effort. There is algolia CLI which technically enables us to do all that (as in exports and imports) but i kinda wanted to deliver a more custom solution, not sure if that was the right choice. Whats your opinion on that @harlan-kohll @kmaid

Reply:
If it covers our usecase then yes I think we should. The less code we write the less we have to maintain